specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,392 @@
+            $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+          inOnderzoek:
+            $ref: '#/components/schemas/VerblijfplaatsBuitenlandInOnderzoek'
+    Verblijfplaats:
+      description: |
+        Gegevens over het verblijf of de woonlocatie van een persoon.
+        * **datumAanvangAdreshuishouding** : de datum van aangifte of ambtshalve melding van verblijf en adres.
+        * **datumIngangGeldigheid** : datum waarop de gegevens over de verblijfplaats geldig zijn geworden.
+        * **datumInschrijvingInGemeente**: bij inschrijving op grond van een verhuisaangifte door de burger is dit de aangiftedatum. Bij inschrijving op grond van een geboorteakte is dit de geboortedatum. Bij ambtshalve inschrijving is dit de datum waarop het voornemen van ambtshalve opneming schriftelijk aan de persoon is medegedeeld.
+        * **datumVestigingInNederland** : datum van inschrijving in Nederland.
+        * **landVanWaarIngeschreven** : het land waar de persoon woonde voor (her)vestiging in Nederland.
+        * **gemeenteVanInschrijving** : de gemeente waar de persoon verblijft en is ingeschreven. De code kan voorloopnullen bevatten."
+      allOf:
+      - $ref : '#/components/schemas/AbstractVerblijfplaats'
+      - type: object
+        properties:
+          functieAdres:
+            $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+          straat:
+            $ref: '#/components/schemas/Straat'
+          huisnummer:
+            $ref: '#/components/schemas/Huisnummer'
+          huisletter:
+            $ref: '#/components/schemas/Huisletter'
+          huisnummertoevoeging:
+            $ref: '#/components/schemas/Huisnummertoevoeging'
+          aanduidingBijHuisnummer:
+            $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+          postcode:
+            $ref: '#/components/schemas/Postcode'
+          adresseerbaarObjectIdentificatie:
+            $ref: '#/components/schemas/AdresseerbaarObjectIdentificatie'
+          nummeraanduidingIdentificatie:
+            $ref: '#/components/schemas/NummeraanduidingIdentificatie'
+          adresregel1:
+            $ref: '#/components/schemas/Adresregel1'
+          adresregel2:
+            $ref: '#/components/schemas/Adresregel2'
+          woonplaats:
+            $ref: '#/components/schemas/Woonplaats'
+          indicatieVestigingVanuitBuitenland:
+            type: boolean
+            description: |
+                          Geeft aan dat de persoon zich vanuit het buitenland heeft ingeschreven.
+          korteNaam:
+            description: |
+                          De officiële openbareruimtenaam uit de Basisregistratie Gebouwen en Adressen (BAG) of een verkorte versie.
+            type: string
+          vanuitVertrokkenOnbekendWaarheen:
```

Comment:
de verwijzing naar #/components/schemas/VertrokkenOnbekendWaarheen' is niet correct. De beschrijving die daarbij hoort is niet de juiste.
Bij vertrokkenOnbekendWaarheen staat "Indicatie dat de ingeschreven persoon is vertrokken naar het buitenland, maar dat niet bekend is waar naar toe."
Bij vanuitVertrokkenOnbekendWaarheen moet staan "Geeft aan dat de persoon is teruggekeerd uit een situatie van 'vertrokken onbekend waarheen'."

Reply:
Aangepast